includes/helper-functions.php

```diff
@@ -0,0 +1,305 @@
+<?php
+/**
+ * Helper functions font settings.
+ *
+ * @package     Bsf_Custom_Fonts
+ * @since       x.x.x
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Return image media ID on basis of file URL.
+ *
+ * @since x.x.x
+ * @return string
+ */
+function bcf_get_media_image_id_by_url( $url ) {
+	global $wpdb;
+	$image = $wpdb->get_col( $wpdb->prepare( "SELECT ID FROM $wpdb->posts WHERE guid='%s';", $url ) );
+	if ( ! empty( $image ) ) {
+		return $image[0];
+	}
+	return false;
+}
+
+/**
+ * Create css for font-face
+ *
+ * @since x.x.x
+ * @param array $font selected font from custom font list.
```

Comment:
@return are not added to the function @imnavanath

Reply:
DONE